include/libsemigroups/sims.hpp

```diff
@@ -339,13 +349,10 @@ namespace libsemigroups {
     //! This function puts a SimsSettings object back into the same state as if
     //! it had been newly default constructed.
     //!
-    //! \parameters (None)
-    //!
     //! \returns A reference to \c this.
     //!
     //! \exception
```

Comment:
```suggestion
    //! \exceptions
```

See #628 for more info.